src/field_theory/fixed.lean

```diff
@@ -210,12 +203,14 @@ begin
     (linear_independent_smul_of_linear_independent G F hs)
 end
 
-instance finite_dimensional_fixed_points : finite_dimensional (fixed_points G F) F :=
+instance : finite_dimensional (fixed_points G F) F :=
 finite_dimensional.finite_dimensional_iff_dim_lt_omega.2 $
-lt_of_le_of_lt (fixed_points.dim_le_card G F) (cardinal.nat_lt_omega _)
+lt_of_le_of_lt (dim_le_card G F) (cardinal.nat_lt_omega _)
+
+protected lemma findim : findim (fixed_points G F) F ≤ fintype.card G :=
```

Comment:
I don't understand the asymmetry between `dim_le_card` and `findim`.

Reply:
Done.